internal/pkg/cli/cmd/template/create.go

```diff
@@ -36,6 +36,9 @@ func EditCommand(depsProvider dependencies.Provider) *cobra.Command {
 	}
 
 	cmd.Flags().SortFlags = true
+	cmd.Flags().String(`id`, ``, "template ID")
+	cmd.Flags().String(`name`, ``, "template name")
+	cmd.Flags().String(`description`, ``, "template description")
```

Comment:
+ chybala moznost zadat tieto hodnoty ako `flag`, ... v dialogu to je, ale chybala definicia flags.